src/redux/slices/authSlice.js

```diff
@@ -19,21 +19,23 @@ export const login = createAsyncThunk("auth/login", async (userData) => {
 
 export const signUp = createAsyncThunk("auth/signup", async (data, thunkAPI) => {
   try {
-   const { firstName, lastName, userName, userphoto, password1 } = data
-   console.log(userName)
+   const { firstName, lastName, username, userphoto, password1 } = data
+   console.log(username)
     const res = await axios.post(
       `/api/auth/signup`,
       {
-        username: userName,
+        username: username,
         firstName: firstName,
         lastName: lastName,
         userphoto: userphoto,
         password1: password1
       },
      
     );
+    console.log(res.data);
     return res.data;
```

Comment:
always remove console while pushing the code

Reply:
removed